.gitignore

```diff
@@ -202,3 +202,6 @@ $RECYCLE.BIN/
 *.lnk
 
 # End of https://www.toptal.com/developers/gitignore/api/windows,node,visualstudiocode,cypressio
+
+cypress/e2e/1-getting-started/*
+cypress/e2e/2-advanced-examples/*
```

Comment:
the lines to not upload the tests that are part of the demo should go in the cypres section of the .gitignore